src/components/molecules/OcTable/OcTable.spec.js

```diff
@@ -206,6 +206,18 @@ describe("OcTable", () => {
     expect(wrapper.html().indexOf("data-file-id")).toBeGreaterThan(-1)
   })
 
+  it("emits contextmenu-clicked event on table row", () => {
+    const wrapper = shallowMount(Table, {
+      propsData: {
+        fields,
+        data,
+        highlighted: [],
+      },
+    })
+    wrapper.vm.$emit("contextmenuClicked")
```

Comment:
I'm not sure if this test makes much sense since you emit and event through test and check if it was committed.

Reply:
True. I think in the past it was on a table row and checked if it bubbled up to the table. That didn't work out and the current state vendor-tests the test utils. 😁

Reply:
Meeeh, let me remove it, then

Reply:
Well found out `wrapper.find(".selector").trigger('contextmenu')` exists, switching to that and triggering on a table row makes the test pass and _actually tests business logic_ ™️